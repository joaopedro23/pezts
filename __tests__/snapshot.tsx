/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react'

import HomePage from '../pages/Home'


it('renders homepage unchanged', () => {
  const { container } = render(<HomePage />)
  expect(container).toMatchSnapshot()
})