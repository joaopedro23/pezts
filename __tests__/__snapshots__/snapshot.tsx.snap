// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="background-image"
  >
    <nav>
      <ul>
        <li />
        <li
          class="Quem-somos"
        >
          <a
            href="/About"
            style="text-decoration: none; color: rgb(0, 0, 0);"
          >
            Quem Somos
          </a>
        </li>
        <li
          class="Agendar-consulta"
        >
          <a
            href="/Agendar"
            style="text-decoration: none; color: rgb(255, 255, 255);"
          >
            Agendar Consulta
          </a>
        </li>
      </ul>
    </nav>
    <div
      class="text-over-image"
    >
      <h1>
        sou home
      </h1>
    </div>
  </div>
</div>
`;
